### ISSUE TITLE
About implement ViT-L as backbone in models

### ISSUE THREAD

**Issue by royalneverwin** [exiawsh/StreamPETR#107]
Hi, recently I have tried to implement ViT-L as backbone like you mentioned in paper to get the best performance.  However, I have experienced some problems. 

To be specific, I intend to use ViT pretrained model from EVA-02, the same as Co-DETR. the link Co-DETR provide is https://github.com/baaivision/EVA/tree/master/EVA-02.  However, it seems that this version of ViT use xformers in Attention module, which causes the following error: Expected query.dim() == 3 to be true, but got false. I try to update my xformers but newer version needs torch version different from mine, so I simply remove xformers and it still works.

**Comment by royalneverwin** [comment]
Another problem is that when I change input image size, it seems that 'rope' module in Attention module is not working anymore, since it needs input image size to have the same H and W size. I just have to simply remove it and it still works. 

When I start to train ViT-L after done things before, it needs 13 days to train 12 epochs on 8*A800 with samples_per_gpu=2. Is it normal to be so slow? Do I have some optional solutions to problems before? Thank you for your time and I look forward to hearing from you!

**Comment by royalneverwin** [comment]
thanks a lot! I will try to fix these problems.
